### PR TITLE
Chapter titles cleanup

### PR DESCRIPTION
**Pull request by wmat**
Cleaned up chapter titles, version numbers, etc.

### CODE REVIEW

File: src/zacas.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-== Atomic Compare-and-Swap (CAS) instructions (Zacas), Version 1.0.0
+== "Cas" and "Zacas" Extensions for Atomic Compare-and-Swap Instructions, Version 1.0.0
```
Comment (ved-rivos) [inline src/zacas.adoc:1]: Should be -  "Zacas" Extension for Atomic Compare-and-Swap (CAS) Instructions, Version 1.0.0

@wmat

File: src/smepmp.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [[smepmp]]
-== PMP Enhancements for memory access and execution prevention on Machine mode (Smepmp)
+== "Smepmp" Enhancements for memory access and execution prevention on Machine mode, Version 1.0.0
```
Comment (aswaterman) [inline src/smepmp.adoc:2]: "Enhancements" should be changed to "Extension" (and it should be singular).

**Review by aswaterman — COMMENTED** [review]
LGTM aside from my one comment.

File: src/smepmp.adoc (hunk 2 above)
Comment (gfavor) [reply src/smepmp.adoc:2]: Note that Smepmp makes enhancements on top of the PMP spec.  So maybe the title should be:

"Smepmp" Extension for PMP enhancements for memory access and execution prevention in Machine mode, Version 1.0.0

Comment (aswaterman) [reply src/smepmp.adoc:2]: Works for me.

Comment (wmat) [reply src/smepmp.adoc:2]: Fixed.  Thanks.

**Pull request merged**